k8s/configuration/config-map.yaml

```diff
@@ -2,35 +2,40 @@
 # This YAML file desribes a ConfigMap for the cluster #
 #######################################################
 
+## Common Env Variables For This Node
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: mdb-mon
+  name: vars
   namespace: default
 data:
-  api-key: "<api key here>"
+  mdb-instance-name: "<name of the mdb instance>"
+  bdb-instance-name: "<name of the bdb instance>"
+  ngx-instance-name: "<name of the ngx instance>"
+  mdb-mon-instance-name: "<name of the mdb monitoring agent instance>"
+  mdb-bak-instance-name: "<name of the mdb backup agent instance>"
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: mdb-backup
+  name: mongodb-whitelist
   namespace: default
 data:
-  api-key: "<api key here>"
+  # We support only 'all' currently
+  allowed-hosts: "all"
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: mdb-fqdn
+  name: bdb-keyring
   namespace: default
 data:
-  fqdn: mdb-instance-0
+  bdb-keyring: "<b64 encoded, ':' separated list of public keys>"
```

Comment:
Nothing in the config-map.yaml file should be b64 encoded. This line and the last line should be edited accordingly.